### DISCUSSION TITLE
Deployment Summaries not expanding?

### DISCUSSION THREAD

**Discussion by musashi-garami** [vercel/community#3718] [Bug Report]
### Question

Hopefully this is the correct spot to ask this.

But I've noticed this quite a lot, the vercel deployment summary (which is very nice) often does not show anything, you press expand and it opens a little and nothing happens... Just seems to hang forever.

**Comment by amyegan** [comment]
Hey @musashi-garami. Would you be able to capture a video the next time this happens and share it here? I have not seen this myself and I want to make sure I dig into the right thing 🙏
